src/components/dialog/Dialog.jsx

```diff
@@ -100,6 +108,20 @@ function BaseDialog({
     }
   }, [open, fireTransitionEndCallbacks]);
 
+  React.useEffect(() => {
+    if (open) {
+      const dialogHeight =
+        containerRef.current &&
+        containerRef.current.getBoundingClientRect().height;
+
+      const overlayHeight =
+        overlayRef.current && overlayRef.current.getBoundingClientRect().height;
```

Comment:
maybe just `overlayRef?.current.getBoundingClientRect().height`?

Reply:
@coderitual `containerRef?.current?.getBoundingClientRect().height;` would work

Reply:
oh sry we probably don't need first condition, so just `containerRef.current?.getBoundingClientRect().height;`